File: mapir-android-service-sdk/src/main/java/ir/map/servicesdk/enums/SelectOptions.java
package ir.map.servicesdk.enums;

public enum SelectOptions {
    PROVINCE("province"),
    CITY("city"),
    COUNTY("county"),
    REGION("region"),
    NEIGHBORHOOD("neighborhood"),
    LAND_USE("landuse"),
    WOOD_WATER("woodwater"),
    ROADS("roads"),
    POI("poi"),
    NEARBY("nearby"),
    NATURAL("natural");

    private String value;

    SelectOptions(String value) {
        this.value = value;
    }

    @Override
    public String toString() {
        return value;
    }
}

File: mapir-android-service-sdk-ktx/src/main/java/ir/map/servicesdk/enums/SelectOptions.kt
package ir.map.servicesdk.enums

enum class SelectOptions(private val value: String) {
    PROVINCE("province"),
    CITY("city"),
    COUNTY("county"),
    REGION("region"),
    NEIGHBOURHOOD("neighborhood"),
    LAND_USE("landuse"),
    WOOD_WATER("woodwater"),
    ROADS("roads"),
    POI("poi"),
    NEARBY("nearby"),
    NATURAL("natural");

    override fun toString() = value
}